devops/ansible/roles/common/tasks/install-tools.yml

```diff
@@ -43,7 +43,6 @@
       - docker-compose
```

Comment:
I think this line is no longer needed

Reply:
This Ansible task will actually uninstall docker-compose. See the later line `state: absent` so the `apt` module will remove it if found.

Reply:
Oh I see, makes sense then 👍